include/unkstruct.h

```diff
@@ -286,3 +286,12 @@ typedef struct {
     s16* coords;
     s16 unk8;
 } Unkstruct_80102CD8;
+
+typedef struct {
+    /* 0x0 */ s16 unk0; // Entity::animSet
```

Comment:
@sonicdcer Should we go ahead and rename the known ones?

Reply:
Yes, it would be alright.